examples/tree_2d_dgsem/elixir_ape_euler_co-rotating_vortex_pair.jl

```diff
@@ -0,0 +1,337 @@
+using OrdinaryDiffEq
+using Trixi
+using LinearAlgebra: norm
+
+module VortexPairSetup
+
+using ForwardDiff
+using LinearAlgebra: norm
+using Trixi
+using Polyester
+using UnPack
+
+
+struct VortexPair{RealT<:Real}
```

Comment:
Add some comment

Reply:
Added a comment describing the setup in faea3d74. This also includes comments for the ICs etc.

Reply:
Looks great, thanks!